db/custom_seeds/notifications.rb

```diff
@@ -0,0 +1,396 @@
+# Notification seed file, to mock notifications for a specific user (ideally yourself).
+# To run the command, run:
+# rails db:seed:notifications USERNAME='your_username'
+# you can also add CLEAN_SETUP="y" if you want to start with fresh notifications.
+
+require "#{Rails.root}/db/seeder"
+
+NUM_TOTAL_STEPS = 11
+
+user = User.find_by(username: ENV["USERNAME"])
+if user.blank?
+  puts "Your user not found. Please ensure the username is present or that the command was properly run: rails db:seed:notifications USERNAME='your_username'"
+  raise StandardError
+end
+
+if ENV["CLEAN_SETUP"].present?
+  puts "starting with none of your existing notifications, followers, articles, and comments..."
+  user.notifications.delete_all
+  Follow.where(followable: user).destroy_all
+  Follow.where(follower: user).destroy_all
+  user.articles.destroy_all
+  user.comments.destroy_all
+end
+
+seeder = Seeder.new
+
+puts "0/#{NUM_TOTAL_STEPS} checking requirements..."
+users = User.where(id: 1..6).count == 6 ? User.where(id: 1..6) : []
+((1..6).to_a - users.pluck(:id)).each do |id|
+  user = User.create!(
+    id: id,
+    name: name,
+    summary: Faker::Lorem.paragraph_by_chars(number: 199, supplemental: false),
+    profile_image: File.open(Rails.root.join("app/assets/images/#{rand(1..40)}.png")),
+    website_url: Faker::Internet.url,
+    twitter_username: Faker::Internet.username(specifier: name),
+    email_comment_notifications: false,
+    email_follower_notifications: false,
+    # Emails limited to 50 characters
+    email: Faker::Internet.email(name: name, separators: "+", domain: Faker::Internet.domain_word.first(20)),
+    confirmed_at: Time.current,
+    password: "password",
+    password_confirmation: "password",
+  )
+  users.push(user)
+end
+
+puts "1/#{NUM_TOTAL_STEPS} seeding broadcast notifications..."
+seeder.create_if_none(Broadcast) do
```

Comment:
This is duplicated from `db/seeds.rb` but I'm not sure where else to put this if we didn't want to duplicate it. I think the next best option is in the new `db/seeder.rb` file as a constant?